config/routes.rb

```diff
@@ -15,6 +15,7 @@
       resources :leads
       resources :campaign_members
       resources :opportunities
+      resources :users
```

Comment:
should this be singular? `user`
since it only responds with data about the current user?

Reply:
@mwharrison Controllers are usually plural. I went with the usual standard.